radis/lbl/factory.py

```diff
@@ -763,13 +813,14 @@ def eq_spectrum(
             raise ValueError(
                 "Tgas should be float or Astropy unit. Got {0}".format(Tgas)
             )
-        self.input.rot_distribution = "boltzmann"  # equilibrium
-        self.input.vib_distribution = "boltzmann"  # equilibrium
+        if not self.input.isatom:
+            self.input.rot_distribution = "boltzmann"  # equilibrium
+            self.input.vib_distribution = "boltzmann"  # equilibrium
```

Comment:
```suggestion
            self.input.rot_distribution = None 
            self.input.vib_distribution = None 
```

Reply:
removed as inapplicable to equilibrium spectra